addons/resource.language.en_gb/resources/strings.po

```diff
@@ -20786,7 +20810,11 @@ msgctxt "#36577"
 msgid "Specify the resolution for the 3D lookup table. Use a lower resolution for quick preview and higher resolution for more accurate picture. Using a high resolution may take seconds to prepare when parameters are changed or a new video is started."
 msgstr ""
 
-#empty string with id 36578
+#. Description of setting "System / Display / Use 10-bit surfaces" with label #36095
+#: system/settings/settings.xml
+msgctxt "#36578"
+msgid "Improves video quality using 10-bit video surfaces. YUV to RGB and limited to full range video conversions are more precise with 10 bits reducing quantization errors and banding. Even if video output is limited to 8-bit (by video driver), improvement's in internal video processing are maintained.[CR][Auto] enables 10 bits only if it's certain that the system supports it, although most modern systems can use it with [Always]. To check effective state use Debug Video Info OSD (Alt+O).[CR]In HDR pass-through mode always are used 10-bit surfaces regardless of this setting."
```

Comment:
If I've understood purpose of the setting correctly then perhaps this could be changed to:  

"Improves video quality by using 10-bit video surfaces for SDR. [CR][Auto] Enables 10 bit for SDR only if the connected display supports HDR. [Always] Enables 10 bit for **SDR** even if the connected display doesn't support HDR. [Never] Do not use 10 bit for SDR.[CR]HDR passthrough mode always uses 10-bit regardless of this setting."

In my view there should just enough to understand what the setting does, no need to go into too much detail.

Reply:
Yes, you have understood correctly and I like the idea of making it clear that it refers only to SDR video.

Somehow I am using the term "video surfaces" to make it clear that it is not about the "video output". That is, what this option does is that internally Kodi processes the video in 10 bits but since the output is SDR in most cases it will end up outputting 8 bits video to the TV since it is the default configuration of Windows / video drivers. Even so, the quality improvement obtained is very notable since the internal processing is much more precise with 10-bit.

Maybe text can be:

Improves video quality by using 10-bit video surfaces for Standard Dynamic Range (SDR) video during internal processing.
[Auto] Enables 10-bit for SDR only if the connected display supports HDR.
[Always] Enables 10-bit for SDR even if the connected display doesn't support HDR.
[Never] Do not use 10-bit for SDR.
HDR passthrough mode always uses 10-bit regardless of this setting.

And perhaps add at end:

Note: the video output to TV may be still limited to 8-bit due the video driver settings or hardware supported modes, but it's still an improvement to use 10-bit internally for video range conversions.